docs/source/guides/contributor/chapters/environment.rst

```diff
@@ -3,14 +3,18 @@ Development environment
 
 .. include:: /.include/helpus.rst
 
-.. warning:: TODO: Needs improvment here. i.e: virtualenvs, GPG, etc.
+.. warning:: TODO: Needs improvement here. i.e: virtualenvs, GPG, etc.
 
 Installing dependencies
 -----------------------
 
 You need to install few dependencies before start coding::
 
- $ sudo dnf install gcc libvirt-devel
+ $ sudo dnf install gcc libvirt-devel python-devel python-enchant python-pip pylint
+
+Optionally, if you want to build the manpage install::
+
+ $ sudo dnf install python-docutils
```

Comment:
Maybe we could add this requirement to `requirements-selftests.txt`

Reply:
pip install the binary as `rst2man.py` and `Makefile` looks for `rst2man`. So this will require a small update in the `Makefile`
to use both.